src/form/v2/birth/forms/child.ts

```diff
@@ -244,17 +244,14 @@ export const childPage = defineFormPage({
     },
     {
       id: 'child.birthLocation',
-      type: 'LOCATION',
+      type: 'LOCATION_SEARCH_INPUT',
```

Comment:
If this is needed, let's make two inputs `FACILITY` and `LOCATION` and so that those inputs themselves have a built in `searchableResource`. This way country config doesn't need to know from where the list is coming from or what data options are even available